charts/mlrun-ce/templates/pipelines/configmaps/workflow-controller-configmap.yaml

```diff
@@ -13,7 +13,9 @@ data:
       secretKeySecret:
         name: mlpipeline-minio-artifact
         key: secretkey
+  {{- if not (include "mlrun-ce.pipelinesV2" .) }}
   containerRuntimeExecutor: emissary
+  {{- end }}
```

Comment:
v2 doesn't need it?

Reply:
v2 doesnt support it - as this is the only option and no other option is available